src/shared/components/plots/ScatterPlotUtils.ts

```diff
@@ -9,3 +9,45 @@ export function getRegressionComputations(
 } {
     return regression.linear(data);
 }
+
+export const makeMultilineAxisLabel = (
+    axisLabel: string | undefined,
+    charLimit: number
+): string => {
+    return !axisLabel
+        ? ''
+        : axisLabel.length > charLimit
+        ? generateLineChunks(axisLabel, charLimit).join('\n')
+        : axisLabel;
```

Comment:
No nested ternaries, please. Also- do you need the inner ternary? What would happen if you did `generateLineChunks(axisLabel, charLimit).join('\n')` when `axisLabel.length <= charLimit`? Wouldn't you just get the original string back? We can afford to waste a few clock cycles here for the sake of code simplicity. I would suggest something like this:

```suggestion
    if (!axisLabel) {
        return ' ';
    }
    
    return generateLineChunks(axisLabel, charLimit).join('\n');
```